gitlabform/gitlab/__init__.py

```diff
@@ -74,3 +75,25 @@ class GitLab(
     GitLabVariables,
 ):
     pass
+
+
+class GitlabWrapper:
+    def __init__(self, gitlabform: GitLab):
+        url = gitlabform.url
+        token = gitlabform.token
+        ssl_verify = gitlabform.ssl_verify
+        timeout = gitlabform.timeout
+        session = gitlabform.session
+
+        self._gitlab: Gitlab = Gitlab(
+            url,
+            token,
+            ssl_verify=ssl_verify,
+            api_version="4",
+            session=session,
+            retry_transient_errors=True,
+            timeout=timeout,
+        )
+
+    def get_gitlab(self):
+        return self._gitlab
```

Comment:
@gdubicki - Not sure how to best structure/setup the python-gitlab so that it can be used going forward. It should facilitate the work of https://github.com/gitlabform/gitlabform/issues/73. Is this the best way to start or do you have other suggestions?

Reply:
@nejch - FYI ☝️

Reply:
Looks good to me!

Reply:
Sorry folks - I was super busy recently and didn't spend much time on GitHub. Looks good to me like this. I think eventually you could get rid of another layer of wrapping here, but looks like the right place to start.

Reply:
Thanks @nejch for the review.

Reply:
I'm going to resolve this thread. Please feel free to re-open it or comment if we need to discuss further. Thanks again.